app/views/content/assessment-only-providers.md

```diff
@@ -502,14 +502,14 @@ keywords:
   - experienced
 ---
 
-If you’re an experienced teacher with a degree, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
+If you have a degree and enough classroom experience, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
```

Comment:
Is this definitely right @tcannon3 - that we want to change from 'experienced' to 'enough classroom experience'?

(Might be the case - just wanted to check that it is accurate!)

Reply:
Yes. According to UCAS and providers, the AO route is for those with substantial classroom experience. The current wording around experienced teachers makes it seem like this is a route for those who are already teachers in which case they would also already have QTS. 
Happy to revise the wording around this though if needed :)

Reply:
@gemmadallmandfe Phoebe has OK'd these changes also